src/payment/payment-strategy-registry.spec.ts

```diff
@@ -47,7 +49,10 @@ describe('PaymentStrategyRegistry', () => {
     class OffsitePaymentStrategy extends BasePaymentStrategy {}
 
     // tslint:disable-next-line:max-classes-per-file
-    class AmazonPayPaymentStrategy extends BasePaymentStrategy {}
+    class AmazonPayPaymentStrategy extends BasePaymentStrategy { }
```

Comment:
Please get rid of the extra space 🙏 
```suggestion
    class AmazonPayPaymentStrategy extends BasePaymentStrategy {}
```

Reply:
done 👍